mathics/doc/latex/sed-hack.sh

```diff
@@ -48,3 +48,11 @@ sed -i -e "s/°/\\\\degree{}/g" documentation.tex
 # from Properties in a Section heading.
 # TODO: figure out how to fix that bug.
 sed -i -e "s/Propertie\\\\/Properties\\\\/g" documentation.tex
+
```

Comment:
This fixes almost all the errors that prevent that mathics.tex from compiling.

Reply:
Note though that for release we need the reorganization started in #984 which is what motivated all of this refactoring.

Reply:
I know, and I am trying to go in that direction, trying to work over something that works.